From 17d015c5180e9e397513569b86b45be7935e7380 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Fri, 12 Oct 2018 13:57:19 +0800
Subject: [PATCH 4829/5242] MLK-19905 ARM64: dts: Enable the busfreq support
 imx8mm ddr4 evk

commit  186c4721e235a31d83359bee46a9ef88238561fc from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable the busfreq support on the i.MX8MM DDR4 evk board.

 BuildInfo:
  - ATF 058a759ffe (imx8m: Add DDR4 DVFS support)

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts     |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts
index cdd4691..5255068 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts
@@ -9,10 +9,6 @@
 
 / {
 	model = "FSL i.MX8MM DDR4 EVK board";
-
-	busfreq {
-		status = "disabled";
-	};
 };
 
 &iomuxc {
-- 
1.7.9.5

